src/main/resources/changesets/failure_table.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+
+    <changeSet author="chrisl" id="failure_table">
+        <createTable tableName="FAILURE">
```

Comment:
While this does not appear to be a reserved word in HSQLDB or MySQL, I'd encourage the use of something less generic in case Cromwell encounters an RDMBS impl that lays claim to `FAILURE`.